### ISSUE TITLE
How to call "scrollToActiveTab" function on external button click.

### ISSUE THREAD

**Issue by AnzilkhaN** [mikejacobson/jquery-bootstrap-scrolling-tabs#68]
I added two custom button for next and prev function. It's working fine. 
On mobile when i click on next button multiple time , **tab head('nav-item')** goes to **"offscreen"**.

Can i call **"scrollToActiveTab"** function when click my custom button ??

**Comment by mikejacobson** [comment]
Yes, you can call the plugin's `scrollToActiveTab` method programmatically:
https://github.com/mikejacobson/jquery-bootstrap-scrolling-tabs#forcing-a-scroll-to-the-active-tab

Keep in mind you might need to wrap that call in a `setTimeout()` to be sure the newly-active tab has been marked as such after being clicked.

**Comment by AnzilkhaN** [comment]
When i try **scrollingTabs('scrollToActiveTab');** on my external button click. I getting an **"error"**.

**"Uncaught ReferenceError: scrollingTabs is not defined"** .  I am not a expert in js. Can you please add a demo for the issue ??

**Comment by AnzilkhaN** [comment]
Hai @mikejacobson , thanks for the demo.

I created a **[plunk](http://plnkr.co/edit/G4bY8i3nofAbSrTKsWAN?p=preview)** to demonstrate my problem.  There is `"prev" & "next"` button in **tabContent** section.

Prev & next are working fine. On mobile `i want to scroll to active tab` when i click on that two button, if the the active **"pill"** not in the viewport.  

If you check my **[plunk demo](http://plnkr.co/edit/G4bY8i3nofAbSrTKsWAN?p=preview)** you can understand my problem.

in _`script.js`_ i added my script, please check.

Please help me to solve this problem. 

Thanks in advance.

**Comment by mikejacobson** [comment]
Thanks for putting that plunk together&mdash;it helps a lot. 

You were so close! 😄  [Here's a fork](http://plnkr.co/edit/sI0xU9VghPnU4qBNDFxM?p=preview) of your plunk showing it working.

In `script.js` I just needed to make two changes:
1. On line 15 your selector was `.nav-tabs` but needed to be `.nav-pills`.
2. On line 67 I added a call to `scrollToActiveTab`&mdash;you just needed to call that after the active tab was changed.
